cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -38,16 +36,16 @@ trait CsvImportExport extends StrictLogging {
                          |    flush-interval = 30m
                          |    shard-mem-size = 256MB
                          |  }""".stripMargin)
-
-    client.setupDataset(dataset,
-                        DatasetResourceSpec(1, 1),
-                        IngestionSource(classOf[CsvStreamFactory].getName, config),
-                        StoreConfig(storeConf)).foreach {
-      case e: ErrorResponse =>
-        println(s"Errors setting up ingestion: $e")
-        exitCode = 2
-        return
-    }
+// FIXME ****** Need to fix this
```

Comment:
Why don't you just get rid of the entire CSV ImportExport thing.  It was for old FiloDB and really doesn't make sense anymore.

Reply:
Especially since you are already doing lots of CLI cleanup.  :)

Reply:
Done